history/history-network.md

```diff
@@ -61,12 +61,8 @@ The history network supports the following protocol messages:
 
 #### `Ping.custom_data` & `Pong.custom_data`
 
-In the history network the `custom_payload` field of the `Ping` and `Pong` messages is the serialization of an SSZ Container specified as `custom_data`:
+In the history network the `custom_payload` field of the `Ping` and `Pong` messages is the serialization of an SSZ Container specified as [Type 2 History Radius Payload](../ping-payload-extensions/extensions/type-2.md)
```

Comment:
Should we not link to type 0 and type 2? Both are ok to be used (if the other side supports it). Or just link to type 0 and mention that capabilities can be checked for further extensions. That text would be more future proof.

Reply:
https://github.com/ethereum/portal-network-specs/pull/348#discussion_r1880762732